packages/remote-feature-flag-controller/src/remote-feature-flag-controller.ts

```diff
@@ -151,6 +158,7 @@ export class RemoteFeatureFlagController extends BaseController<
     this.#disabled = disabled;
     this.#clientConfigApiService = clientConfigApiService;
     this.#getMetaMetricsId = getMetaMetricsId;
+    this.#clientVersion = clientVersion;
```

Comment:
To whoever implements this in the clients, remember to remove the `-flask.0` parts of the version as it causes problems when comparing semantic versions

Reply:
The `isValidSemVerVersion` function is already easily accessible from clients, it's from `@metamask/utils`